examples/helia-remix/.eslintrc.cjs

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
we should make sure we are extending the styles used in the rest of the helia repos.. you should be able to extend 'ipfs' and get the lint rules from https://github.com/ipfs/eslint-config-ipfs

Reply:
Okay